src/_pytest/logging.py

```diff
@@ -297,6 +297,13 @@ def add_option_ini(option, dest, default=None, type=None, **kwargs):
         default=None,
         help="Auto-indent multiline messages passed to the logging module. Accepts true|on, false|off or an integer.",
     )
+    group.addoption(
```

Comment:
Why is the new option just a command line option and not an `ini` option? I guess that it can't be specified in e.g. `pytest.ini`, right?

Reply:
It is possible to use [`addopts`](https://docs.pytest.org/en/stable/reference/reference.html#confval-addopts) to set that for every invocation if one chooses, is that what you are asking?

Reply:
Yes, kind of. In the logging plugin we have the `add_option_ini` function which is used for most parameters of the logging plugin. I think that whenever we don't use this function, we should explain in a comment why we don't use it. Even better would be an explanation in the docstring of `pytest_addoption` in logging.py that explains when if new options should be added as ini options, as cli options or as both.

Reply:
There's still time to change this, as 7.2.0 was not released yet. 😁